### ISSUE TITLE
drush pm:security reports Drupal core 8.7.0-rc1 as suggested version

### ISSUE THREAD

**Issue by timwood** [drush-ops/drush#4053]
**Describe the bug**
When running `drush pm:security` drupal/core is reported as having an outstanding security update.
```
 [warning] One or more of your dependencies has an outstanding security update. Please apply update(s) immediately.
 [notice] Try running: composer require drupal/core:^8.7.0-rc1 --update-with-dependencies
 [notice] If that fails due to a conflict then you must update one or more root dependencies.
+-------------+-------------------+-------------------+
| Name        | Installed Version | Suggested version |
+-------------+-------------------+-------------------+
| drupal/core | 8.6.15            | 8.7.0-rc1         |
+-------------+-------------------+-------------------+
```

**To Reproduce**
Ran `drush pm:security`

**Expected behavior**
Drush to report no security update available.

**Actual behavior**
Drush reported a security update even though I have version 8.6.15.

**Workaround**
N/A

### System Configuration
| Q               | A
| --------------- | ---
| Drush version?  | 9.4.0
| Drupal version? | 8.6.15
| PHP version     | 7.2
| OS?             | Linux

**Comment by joestewart** [comment]
Is it https://github.com/drush-ops/drush/issues/3748? If so that was fixed in 9.6.0

**Comment by weitzman** [comment]
Same bug as #4050.

Also, please upgrade to latest Drush release before reporting bugs.
